### PR TITLE
Adds xcm channels guide

### PR DESCRIPTION
**Pull request by augustocollado**
### Description

This PR adds the XCM channels management guide.
(This guide will benefit greatly once we have the xcm learn article to provide the basic concepts)

Known debts:
- update screenshots
- Add references to the learn article when referring to concepts 
- Retrieving sudo / registration account must be another guide or snippet

### Checklist

- [ ] I have added a label to this PR 🏷️
- [ ] I have run my changes through Grammarly
- [ ] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

**Review by albertov19 — CHANGES_REQUESTED** [review]
Left some feedback. I'm a bit unsure on the last sections as they feel empty and disconnected to the image where you show the different sections

File: builders/manage/dapp/xcm-channels.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+The panel will show your appchain's communication channels status along with several available actions. The elements you are presented with are:
+
+- Sovereign account
+- Incoming/Outgoing channel requests - 
+- Established channels
+- Request to open new channels
+
+![The channel management panel](/images/builders/manage/dapp/xcm-channels/xcm-channels-4.webp)
+
+### Sovereign Account {: #sovereign-account }
```
[the remark this answers is not in the document]
Comment (augustocollado) [reply builders/manage/dapp/xcm-channels.md:70]: Being zero, I'll just remove the "the tanssi team will help you obtain the tokens", and state the active configuration.
I think we could add a table once we add mainnet settings

File: builders/manage/dapp/xcm-channels.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+!!! note
+    The appchain's registration account is always a Substrate one, whereas the appchain's Sudo account depends on the chain type. If the chain is EVM-compatible, then the Sudo account will be an Ethereum type and, otherwise, a Substrate type.
+
+### Retrieving the Registration Account {: #retrieving-registration-account }
+
+If you're unsure what your registration account is, you can query it directly from the Tanssi orchestrator chain, which keeps records of every registered appchain. To do so, head to the **Chain state** section on the Polkadot.js Apps connected to the orchestrator chain for [snap appchains](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-flashbox-rpc.a.stagenet.tanssi.network#/chainstate){target=\_blank} or [dedicated appchains](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/chainstate){target=\_blank}, and take the following steps:
+
+1. Select the **registrar** storage module
+2. Select **registrarDeposit**
+3. Insert your appchain id
```
Comment (eshaben) [inline builders/manage/dapp/xcm-channels.md:33]:
```suggestion
3. Insert your appchain ID
```

File: builders/manage/dapp/xcm-channels.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+### Sovereign Account {: #sovereign-account }
+
+Before opening a new channel, the appchain's sovereign account in the relay chain must be funded with enough tokens to be locked as a channel deposit.
+
+A sovereign account is a keyless account that belongs to the appchain in a different consensus system, the relay chain in this case. It can only be used by the appchain's governor.
+
+In this section, you can see your appchain's sovereign account balance, copy its address, and deposit tokens.
+
+!!! note
+    Your appchain's sovereign account in the relay chain must be funded with relay chain tokens. On Dancebox, the relay chain uses UNIT tokens, which the Tanssi team will help you obtain when needed.
```
Comment (eshaben) [inline builders/manage/dapp/xcm-channels.md]: how can they get ahold of the team?

File: builders/manage/dapp/xcm-channels.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+3. **Established channels** - Once the destination chain's governor has accepted the channel request, the channel becomes open and available for message transmission.
+
+    In this section, you can see the list of accepted channels your appchain has, the direction in which the messages flow through the channel, and cancel the channel
+
+4. **Request to open new channels** - this option allows you to select an existing appchain within the ecosystem and request a channel to be opened. The [next section](#request-new-channel) explains how to do it
+
+![The channel management panel](/images/builders/manage/dapp/xcm-channels/xcm-channels-4.webp)
+
+## Request to open new channel {: #request-new-channel }
```
Comment (eshaben) [inline builders/manage/dapp/xcm-channels.md:84]:
```suggestion
## Request to Open New Channel {: #request-new-channel }
```

File: builders/manage/dapp/xcm-channels.md (hunk 3 above)
Comment (augustocollado) [reply builders/manage/dapp/xcm-channels.md]: I changed this wording a bit, now it says "[...] On Dancebox, the relay chain uses UNIT tokens, and the required amount is currently configured to zero."

**Review by albertov19 — APPROVED** [review]

File: builders/manage/dapp/xcm-channels.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+description: Tanssi appchains benefit from native cross-chain communication, which allows fast and secure bridging leveraging the architecture they are built on top of.
+---
+
+# Manage Cross-Chain Communication Channels
+
+## Introduction {: #introduction }
+
+All the Tanssi appchains have an inherent capability to communicate and interoperate with any other appchain from the ecosystem. This native cross-chain communication feature is possible thanks to the unique infrastructure the appchains are built on top of, leveraging the Cross-Consensus Message format (XCM for short), which facilitates communication between different consensus systems.
+
+The first step to enable communication between appchains is opening a channel. The process of opening a channel starts by sending a request to the appchain you want to establish communications with. Once the request has been accepted by the destination chain's governor, a channel will be opened.
```
[the remark this answers is not in the document]
Comment (augustocollado) [reply builders/manage/dapp/xcm-channels.md:12]: Yes, unfortunately, we don't yet have that article. It's a known shortcoming, stated in the pr's description

**Pull request merged**